src/hevm/src/EVM/SymExec.hs

```diff
@@ -369,6 +371,31 @@ simplify e = if (mapExpr go e == e)
       | otherwise = o
     go o@(ReadWord (Lit _) _) = Expr.simplifyReads o
     go o@(ReadByte (Lit _) _) = Expr.simplifyReads o
+    go (WriteWord (Lit locLit) (Lit valLit) (ConcreteBuf buf)) = ConcreteBuf (BS.pack $ begin ++ middle ++ end)
```

Comment:
Could we reuse `writeWord`, `writeByte` and `copySlice` from Expr.hs here?

Reply:
Oh damn, good point, I didn't notice these...